src/deterministic_vi/elbo_objective.jl

```diff
@@ -315,7 +361,26 @@ function accumulate_source_brightness!{NumType <: Number}(
         end
     end
 
-    calculate_var_G_s!(elbo_vars, active_source)
+    calculate_var_G_s!(elbo_vars, E_G_s, E_G2_s, var_G_s, is_active_source)
+end
+
+
+function calculate_source_pixel_brightness!{NumType <: Number}(
+                    elbo_vars::ElboIntermediateVariables{NumType},
+                    ea::ElboArgs{NumType},
+                    sbs::Vector{SourceBrightness{NumType}},
+                    s::Int, b::Int)
+
+    calculate_source_pixel_brightness!(
+        elbo_vars,
+        ea,
+        elbo_vars.E_G_s,
+        elbo_vars.var_G_s,
+        elbo_vars.fs0m_vec[s],
+        elbo_vars.fs1m_vec[s],
+        sbs[s],
+        b, s,
+        s in ea.active_sources)
 end
```

Comment:
Doesn't seem worth it to me, unless the wrapper is called from a lot of different places. Extra indirection is confusing, especially for someone who didn't write the code. But as you wish.